lib/change_stream.js

```diff
@@ -556,8 +525,7 @@ function processNewChange(args) {
     );
 
     if (eventEmitter) return changeStream.emit('error', noResumeTokenError);
-    if (typeof callback === 'function') return callback(noResumeTokenError, null);
-    return changeStream.promiseLibrary.reject(noResumeTokenError);
+    return callback(noResumeTokenError, null);
```

Comment:
nit since we're here anyway: `return callback(noResumeTokenError);` (no `null`). This also applies to a number of these callbacks here.